lib/ret_web/controllers/api/v1/account_controller.ex

```diff
@@ -0,0 +1,34 @@
+defmodule RetWeb.Api.V1.AccountController do
+  use RetWeb, :controller
+  import RetWeb.ApiHelpers
+
+  alias Ret.{Account}
+  alias RetWeb.Api.V1.{AccountView}
+
+  # TODO move to a file
```

Comment:
Remove/resolve TODO

Reply:
this was also longer term (ie when we want to have swagger docs, etc). will remove